### DISCUSSION TITLE
dhaqsoPay logo

### DISCUSSION THREAD

**Discussion by hayle01** [dugsiiyeinc/dhaqsoPay#10] [dhaqsoPay Extension Developers]
Eng @sharafdin  waxaan rabaa logo-da, Fonts iyo colorada project-ga

**Comment by shiinedev** [comment]
@Hayle01 bro logada waxa ka heleysa repositoriga dhexdiisa folderka  .github ey ku qorantahay aye ku jirta

**Reply by hayle01** [reply]
> @Hayle01 bro logada waxa ka heleysa repositoriga dhexdiisa folderka .github ey ku qorantahay aye ku jirta
Thank you sxb waan arkay

**Comment by sharafdin** [comment]
@Hayle01 eng logo waaba sidaas. mida kale icons waan kuso daray isticmaal midka 128 ka ah. colors-kana waa kan:
primary: #222161
secondary: #3abfef
